Packs/iDefense/Integrations/iDefense_v2/iDefense_v2.py

```diff
@@ -276,7 +284,7 @@ def uuid_command(client: Client, args: dict, reliability: DBotScoreReliability)
     """
     uuid: str = str(args.get('uuid'))
     try:
-        res = client.threat_indicator_search(url_suffix=f'/{uuid}')
+        res = client.threat_indicator_search(url_suffix=f'/v0/{uuid}')
     except Exception as e:
```

Comment:
Please raise the exception rather than `return CommandResults`, so users know the command failed

Reply:
ok let me look into that

Reply:
this part was coded by PANW, and it works fine, it will display the error to the user if something happens.
 
![image](https://user-images.githubusercontent.com/40510780/146357027-d494862a-c66d-4f0c-86a8-9fbb3d6a600b.png)

Reply:
What if it's being run as part of a playbook?

Reply:
similar error should show up as it will ultimately end up calling uuid command

Reply:
it will be ultimately calling uuid command at the end, so the same result will come as the above

Reply:
Here too, if you can narrow down the `except Exception` to a specific exception this may prevent bugs in the future.

Reply:
https://xsoar.pan.dev/docs/integrations/code-conventions#exceptions-and-errors

Reply:
When empty results are received, do `return_results` with an empty object (like rows 290-291 before the change), unless an empty result is an error that should halt playbooks.

On an error that should halt playbooks `raise e`, and `main()` will handle it and print the error.

Reply:
i think earlier the code described what you are mentioning . so probably I will get back those lines of code 👍

Reply:
but it was without `return_results`